### PR TITLE
EN-2313 DE ratings

### PR DESCRIPTION
**Pull request by rzsouza**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/main/java/org/entando/entando/aps/system/services/digitalexchange/rating/DERatingsSummary.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+package org.entando.entando.aps.system.services.digitalexchange.rating;
+
+import java.util.Objects;
+
+public class DERatingsSummary {
+
+    private String componentId;
+    private int rating;
+    private int numberOfInstalls;
```
[the remark this answers is not in the document]
Comment (rzsouza) [reply src/main/java/org/entando/entando/aps/system/services/digitalexchange/rating/DERatingsSummary.java]: Removed this

**Comment by entando-service** [comment]
All is well

**Pull request merged**